util/chplenv/third_party_utils.py

```diff
@@ -25,8 +25,13 @@ def default_uniq_cfg_path():
 # Returns the path to the packages install directory
 #
 @memoize
-def get_cfg_install_path(pkg, ucp=default_uniq_cfg_path()):
-    return os.path.join(get_chpl_third_party(), pkg, 'install', ucp)
+def get_cfg_install_path(pkg, ucp=default_uniq_cfg_path(), host_or_target=None):
+    if host_or_target is None:
+        return os.path.join(get_chpl_third_party(), pkg, 'install', ucp)
+    else:
+        # some third party packages (like jemalloc) can be built for both host and target
+        # in which case they are under jemalloc/install/{host,target}/... for example
+        return os.path.join(get_chpl_third_party(), pkg, 'install', host_or_target, ucp)
```

Comment:
Seems like this could be slightly simpler if `host_or_target` defaulted to `''` instead of `None` and we just always did the join with that?

Reply:
TIL os.path.join removes empty string. I'm happy with that approach. Thanks for the suggestion